### PR TITLE
Changed message width.

### PR DESCRIPTION
**Pull request by dmitry-zezik**
Changed message width from 30 up to 250 letters. Added ... at the end of wisible part

### CODE REVIEW

File: src/qml/pages/MainPage.qml [hunk 1]
```diff
@@ -175,4 +202,15 @@ Page {
             controllers.fileCloud.addFile(url)
         }
     }
+
+    function textClipper(text, textPlaceholderWidth, textContentWidth) {
```
Comment (fpohtmeh-github) [inline src/qml/pages/MainPage.qml:206]: This fix looks complicated.
Can it be done without extra function, just with pure text eliding & anchors?

Comment (dmitry-zezik) [reply src/qml/pages/MainPage.qml:206]: This fix adds functionality to trim a string and add three dots to the end of a string. There is no simpler method. I can try to make the code more concise.

Comment (fpohtmeh-github) [reply src/qml/pages/MainPage.qml:206]: So why text eliding can't be used instead of this?

Comment (dmitry-zezik) [reply src/qml/pages/MainPage.qml:206]: My bad. I'll fix

File: src/qml/pages/MainPage.qml [hunk 2]
```diff
@@ -52,7 +52,7 @@ Page {
 
                 Column {
                     Layout.fillWidth: true
-                    clip: true
+                    clip: false
```
Comment (fpohtmeh-github) [inline src/qml/pages/MainPage.qml:55]: Clip has default value, remove this line

File: src/qml/pages/MainPage.qml [hunk 3]
```diff
@@ -61,13 +61,16 @@ Page {
...
                     Text {
+                        id: messageBody
                         color: Theme.secondaryTextColor
                         font.pointSize: UiHelper.fixFontSz(12)
-                        text: model.lastMessageBody
                         width: parent.width
+                        text: model.lastMessageBody
                         elide: Text.ElideRight
-                        textFormat: Text.RichText
+//                        textFormat: Text.RichText
```
Comment (fpohtmeh-github) [inline src/qml/pages/MainPage.qml:70]: Remove commented code

File: src/qml/pages/MainPage.qml [hunk 4]
```diff
@@ -61,13 +61,16 @@ Page {
...
+                        maximumLineCount: 1
```
Comment (fpohtmeh-github) [inline src/qml/pages/MainPage.qml:71]: I don't know why this was added. Works fine without it for me.
For multi-line messages Qt replaces \n with space, it's fine.
Remove if you don't have the case there is a difference

File: src/qml/pages/MainPage.qml [hunk 5]
```diff
@@ -61,13 +61,16 @@ Page {
...
                     }
+
```
Comment (fpohtmeh-github) [inline src/qml/pages/MainPage.qml:73]: There is no need to add extra new line between 2 closing brackets. Seems it left after removing of textClipper().
That's why sometimes it's better to undo commits by git, not manually.

File: src/qml/pages/MainPage.qml [hunk 6]
```diff
@@ -88,6 +91,7 @@ Page {
                 }
 
                 onClicked: controllers.chats.openChatById(model.id)
+
```
Comment (fpohtmeh-github) [inline src/qml/pages/MainPage.qml:94]: remove \n

File: src/qml/pages/MainPage.qml (hunk 2 above)
Comment (dmitry-zezik) [reply src/qml/pages/MainPage.qml:55]: It was TRUE previously

File: src/qml/pages/MainPage.qml (hunk 3 above)
Comment (dmitry-zezik) [reply src/qml/pages/MainPage.qml:70]: Done

File: src/qml/pages/MainPage.qml (hunk 4 above)
Comment (dmitry-zezik) [reply src/qml/pages/MainPage.qml:71]: Removed

File: src/qml/pages/MainPage.qml (hunk 5 above)
Comment (dmitry-zezik) [reply src/qml/pages/MainPage.qml:73]: Removed

File: src/qml/pages/MainPage.qml (hunk 6 above)
Comment (dmitry-zezik) [reply src/qml/pages/MainPage.qml:94]: Removed

**Review by fpohtmeh-github — APPROVED** [review]

**Pull request merged**